build/make/dependencies-glide.mk

```diff
@@ -9,10 +11,10 @@ ifeq ($(ENVIRONMENT), ci)
 endif
 
 .PHONY: update-dependencies
-update-dependencies: $(GLIDE)
+update-dependencies: $(GLIDE) ## Install glide
 
 .PHONY: dependencies
-dependencies: vendor
+dependencies: vendor ## Install dependencies using glide
```

Comment:
see above

Reply:
There is no `locally` here, what do you mean?